src/symbols/analyzer.rs

```diff
@@ -181,55 +194,64 @@ impl<'a> RootSymbol<'a> {
         .map(|(k, v)| (k, v.0.into_inner()))
         .collect(),
     };
-    Some(self.finalize_insert(ModuleSymbol::Esm(module_symbol)))
+    Some(self.finalize_insert(ModuleInfo::Esm(module_symbol)))
   }
 
-  fn analyze_json_module(&self, json_module: &JsonModule) -> ModuleSymbolRef {
+  fn analyze_json_module(&self, json_module: &JsonModule) -> ModuleInfoRef {
     let specifier = &json_module.specifier;
     // it's not ideal having to use SourceTextInfo here, but it makes
     // it easier to interop with ParsedSource
     let source_text_info = SourceTextInfo::new(json_module.source.clone());
     let range = source_text_info.range();
-    let module_id = ModuleId(self.ids_to_symbols.len() as u32);
-    let module_symbol = JsonModuleSymbol {
+    let module_id = ModuleId(self.ids_to_modules.len() as u32);
+    let decls = {
+      let range = {
+        let source = source_text_info.text_str();
+        let start_whitespace_len = source.len() - source.trim_start().len();
+        let end_whitespace_len = source.len() - source.trim_end().len();
+        SourceRange::new(
+          range.start + start_whitespace_len,
+          range.end - end_whitespace_len,
+        )
+      };
```

Comment:
I think I saw the same code before in one of the PRs - maybe it should be a helper function?

Reply:
It only exists here. It was shifted from down below.